am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/Permissions.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.amlib.enums;
+
+import java.util.List;
+
+public enum Permissions {
+    HIDE(0),
+    CREATE(1),
+    READ(2),
+    UPDATE(4),
+    SHARE(8),
+    DELETE(16)
+    ;
+
+    private int value;
+
+    Permissions(int value) {
+        this.value = value;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public static int sumOf(List<Permissions> permissions) {
+        int sum = 0;
+
+        for(Permissions permission: permissions) {
+            sum += permission.getValue();
```

Comment:
shouldn't you here use *&=* instead of *+=* ??
I'd imagine that if I would pass .Create and .Read cases, it will produce 000011. 
With current implementation when you pass .Hide and .Create it will generate 000001 - which is wrong?

Reply:
This += is a regular arythmetic operation that adds up integers, it's not a binary one. .Create + .Read = 1 + 2 = 3